src/credentials/DeviceAttestationVerifier.cpp

```diff
@@ -54,17 +54,12 @@ class UnimplementedDACVerifier : public DeviceAttestationVerifier
     }
 
     AttestationVerificationResult ValidateCertificateDeclarationPayload(const ByteSpan & certDeclBuffer,
-                                                                        const ByteSpan & firmwareInfo, uint16_t clusterVendorId,
-                                                                        uint16_t clusterProductId, uint16_t dacVendorId,
-                                                                        uint16_t dacProductId, uint16_t paiProductId) override
+                                                                        const ByteSpan & firmwareInfo,
+                                                                        DeviceInfoForAttestation deviceInfo) override
```

Comment:
Suggest `const DeviceInfoForAttestation &` in the signature.

Reply:
Fixed in 9cddb31